nilearn/connectome/connectivity_matrices.py

```diff
@@ -206,15 +207,26 @@ def sym_to_vec(symmetric):
     symmetric : numpy.ndarray, shape (..., n_features, n_features)
         Input array.
 
+    keep_diagonal : boolean, optional
+        Whether to keep or not the coefficient in the diagonal of the matrix.
+        Default is True.
+
     Returns
     -------
-    output : numpy.ndarray, shape (..., n_features * (n_features + 1) / 2)
-        The output flattened lower triangular part of symmetric.
+    output : numpy.ndarray
+        The output flattened lower triangular part of symmetric. Shape is
+        (..., n_features * (n_features + 1) / 2) if keep_diagonal is True and
+        (..., (n_features - 1) * n_features / 2) if it is False.
+
+
     """
-    scaling = sqrt(2) * np.ones(symmetric.shape[-2:])
-    np.fill_diagonal(scaling, 1.)
-    tril_mask = np.tril(np.ones(symmetric.shape[-2:])).astype(np.bool)
-    return symmetric[..., tril_mask] * scaling[tril_mask]
+    if keep_diagonal:
+        scaling = sqrt(2) * np.ones(symmetric.shape[-2:])
```

Comment:
I proprose to remove this scaling, which is horribly counter-intuitive. 
It might be available under an 'isometric' option if there is any reason to keep it, but I don't  see any use case.